src/Uno.UI.Runtime.Skia.Gtk/GTK/GtkHost.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Foundation.Extensibility;
+using Windows.UI.Xaml;
+using WUX = Windows.UI.Xaml;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class GtkHost
+	{
+		private Func<WUX.Application> _appBuilder;
+		private static Gtk.Window _window;
+
+		public static Gtk.Window Window => _window;
+
+		public GtkHost(Func<WUX.Application> appBuilder, string[] args)
+		{
+			_appBuilder = appBuilder;
+		}
+
+		public void Run()
+		{
+			Gtk.Application.Init();
+
+			ApiExtensibility.Register(typeof(Windows.UI.Core.ICoreWindowExtension), o => new GtkUIElementPointersSupport(o));
+
+			_window = new Gtk.Window("Uno Host");
+			_window.SetDefaultSize(1024, 800);
+			_window.SetPosition(Gtk.WindowPosition.Center);
+
+			_window.DeleteEvent += delegate
+			{
+				Gtk.Application.Quit();
+			};
+
+			Windows.UI.Core.CoreDispatcher.DispatchOverride
+			   = d =>
+			   {
+				   if (Gtk.Application.EventsPending())
+				   {
+					   Gtk.Application.RunIteration(false);
+				   }
+
+				   GLib.Idle.Add(delegate
+				   {
+					   Console.WriteLine("iteration");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)